### PR TITLE
Add 4th of July Event items

### PR DESCRIPTION
**Pull request by ZethalMC**
Some items are missing textures. They'll be added as soon as I have them finished.

### CODE REVIEW

**Comment by ZethalMC** [comment]
I am just bad at using separators 😄

**Comment by ZethalMC** [comment]
Finally! Have fun!

**Comment by orange-bazookas** [comment]
the items have way too high sell values for their rarity, and imo you could have just used last year's 4th of july items instead of making new ones

**Comment by orange-bazookas** [comment]
it also seems not all items are mine&fish, is this a bug?

**Comment by ZethalMC** [comment]
> the items have way too high sell values for their rarity, and imo you could have just used last year's 4th of july items instead of making new ones

This is fixed right away.

**Comment by ZethalMC** [comment]
> it also seems not all items are mine&fish, is this a bug?

This is not a bug, just to make the items a bit more random. Some normal items can't be fished or mined too, as they're specific to one method.

File: common/data/data.json [hunk 1]
```diff
@@ -1446,6 +1560,24 @@
...
+    "yellow_firework_rocket": "<:yellow_firework_rocket:1256238590640525444>",
+    "american_flag": "<:american_flag:1256238578187370497>",
+    "lady_liberty_torch": "<:air:828359049602662431>",
+    "declaration_of_independence": "<:air:828359049602662431>",
+    "uncle_sams_hat": "<:uncle_sams_hat:1256238589218525264>",
+    "freedom_firework_bundle": "<:air:828359049602662431>",
+    "lady_liberty_tabula_ansata": "<:air:828359049602662431>",
+    "hot_dog": "<:hot_dog:1256238582985785470>",
+    "4th_of_july_decorations": "<:air:828359049602662431>",
+    "a_literal_bonfire": "<:air:828359049602662431>",
```
Comment (Iapetus-11) [inline common/data/data.json:1577]: For items with no emoji just don't add them here ig

**Review by Iapetus-11 — CHANGES_REQUESTED** [review]
I think a few should be removed:
- Eating Competition Prize
- A Literal Bonfire
- 4th Of July Decorations
- Lady Liberty Tabula Ansata
- Lady Liberty Torch
- Declaration Of Independence
- Freedom Firework Bundle

But I do like the rest, we could also maybe add:
- Komodo 30000 SuperNova Aerial Firework (with a bomb emoji)

**Comment by Iapetus-11** [comment]
@ZethalMC Since you opened this from your `main` branch I can't push changes to it...
image

I'm going to close this PR and make a new one off your branch.

**Pull request closed without merging**